bin/gltf-pipeline.js

```diff
@@ -17,10 +18,16 @@ if (process.argv.length < 3 || defined(argv.h) || defined(argv.help)) {
         '  -s --separate, writes out separate geometry/animation data files, shader files and textures instead of embedding them in the glTF file.\n' +
         '  -t --separateImage, write out separate textures, but embed geometry/animation data files, and shader files.\n' +
         '  -q, quantize the attributes of this model.\n' +
-        '  --ao.diffuse, bake ambient occlusion into the diffuse texture. Defaults to false.\n' +
+        '  --ao.vertex, bake ambient occlusion to vertices instead of the diffuse texture and modify the shader. Defaults to false.\n' +
```

Comment:
Exposing all these options is fine for the JavaScript/Node API and for advanced command-line users, but most of our users will not know how to tweak these.  They will understand:
* AO on or off
* Some type of scale for "slower and nice looking" and "faster and OK looking"
* Ground plane on or off

Is there a way to translate these user-friendly parameters to all the potential options?  Are any options not important enough to expose?  Can any of these options be generated based on the model, e.g., `density` based on the bounding box?